.github/workflows/build_pr_documentation.yml

```diff
@@ -14,5 +14,5 @@ jobs:
       commit_sha: ${{ github.event.pull_request.head.sha }}
       pr_number: ${{ github.event.number }}
       package: transformers
-      languages: de en es fr hi it ko pt tr zh ja te
+      languages: de en es fr hi it ko pt tr zh ja te ar
```

Comment:
Can you add Arabic in its alphabetic order?

Reply:
Done